profiles/common/modules/features/multisite_maps/multisite_maps.module

```diff
@@ -86,15 +86,17 @@ function escape($value) {
  */
 function multisite_maps_get_library_path() {
   global $conf;
-  $path = 'sites/all/libraries/FusionMaps XT/';
-  $lpath = libraries_get_path('FusionMaps XT/');
+
   if (isset($conf['fusion_maps_path'])) {
-    $path = $conf['fusion_maps_path'];
+    return $conf['fusion_maps_path'];
+  }
+  if (module_exists('libraries')) {
+    $path = libraries_get_path('FusionMaps XT');
   }
-  elseif ($lpath) {
-    $path = $lpath . '/';
+  if (!$path) {
+    $path = 'profiles/common/libraries/FusionMaps XT';
   }
-  return $path;
+  return $path . '/';
```

Comment:
It's better to add the slash on the line 50:
``include_once multisite_maps_get_library_path() . '/Code/PHP/Includes/FusionMaps.php';``
instead of 
``include_once multisite_maps_get_library_path() . 'Code/PHP/Includes/FusionMaps.php';``